config/secrets.yml

```diff
@@ -12,11 +12,14 @@
 
 development:
   secret_key_base: 79b6fed857102a03cf6815ba94a75a1984cb4b14c9a9b72a472ca53596f5b96bae4472cfd74f270e09288dcde1ca96b937d37c97beefc596fa9f3878be2ffbf0
-
+  email: cloneyislandmailer@gmail.com
+  password: PassWord1234!
```

Comment:
Let's make sure we get these on production so we can make this work for our final client meeting.